ingest/entryWriter.go

```diff
@@ -35,13 +35,14 @@ const (
 
 	MAX_UNCONFIRMED_COUNT int = 1024 * 4
 
-	MINIMUM_TAG_RENEGOTIATE_VERSION uint16        = 0x2 // minimum server version to renegotiate tags
-	MINIMUM_ID_VERSION              uint16        = 0x3 // minimum server version to send ID info
-	MINIMUM_INGEST_OK_VERSION       uint16        = 0x4 // minimum server version to ask
-	MINIMUM_DYN_CONFIG_VERSION      uint16        = 0x5 // minimum server version to send dynamic config block
-	MINIMUM_INGEST_STATE_VERSION    uint16        = 0x6 // minimum server version to send detailed ingester state messages
-	MINIMUM_INGEST_EV_VERSION       uint16        = 0x8 // minimum server version to send enumerated values attached to entries
-	maxThrottleDur                  time.Duration = 5 * time.Second
+	MINIMUM_TAG_RENEGOTIATE_VERSION uint16 = 0x2 // minimum server version to renegotiate tags
+	MINIMUM_ID_VERSION              uint16 = 0x3 // minimum server version to send ID info
+	MINIMUM_INGEST_OK_VERSION       uint16 = 0x4 // minimum server version to ask
+	MINIMUM_DYN_CONFIG_VERSION      uint16 = 0x5 // minimum server version to send dynamic config block
+	MINIMUM_INGEST_STATE_VERSION    uint16 = 0x6 // minimum server version to send detailed ingester state messages
+	MINIMUM_INGEST_EV_VERSION       uint16 = 0x8 // minimum server version to send enumerated values attached to entries
```

Comment:
we need to bump the version and have the MINIMUM_DITTO_VERSION sent so that during negotiation the ditto client can refuse to connect